modules/administration/pages/troubleshooting/tshoot-container-full-disk.adoc

```diff
@@ -24,24 +24,31 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 In case a dedicated disk mounted as a persistent storage medium of a container runs out of storage space, an emergency action is needed.
 
 Proceed as follows to solve the problem with resizing the storage medium.
+Run all listed commands as [literal]``root`` on the container host.
 
 .Procedure: Resize Storage Medium
 
 . Increase the disk (for example, [path]``/dev/vdb``) size.
   For example, if the disk is a QCOW2 image, use the command [command]``qemu-img resize``.
 
-. Ensure the partition uses all available space:
-* open a shell in the container using [command]``mgrctl term``.
-* [command]``sudo parted /dev/vdb``
-* [literal]``(parted)`` [command]``print``
-* [literal]``(parted)`` [command]``resizepart NUMBER 100%`` where [literal]``NUMBER`` is the partition number shown by [command]``print`` command (for example, [literal]``1`` if [literal]``/dev/vdb1``)
-* [literal]``(parted)`` [command]``quit``
+. Ensure the partition uses all available space.
+  Run the following commands:
+
+--
+.. [command]``parted /dev/vdb``
+.. [literal]``(parted)`` [command]``print``
+.. [literal]``(parted)`` [command]``resizepart NUMBER 100%`` where [literal]``NUMBER`` is the partition number shown by [command]``print`` command (for example, [literal]``1`` if [literal]``/dev/vdb1``)
+.. [literal]``(parted)`` [command]``quit``
```

Comment:
This is again more complicated by our making 😄

Initial `mgr-storage-server` created partition so this procedure works.

However later it was modified (to support LVM) that it does not have any partition, so this step can be skipped.

So the procedure:
- is there a partition -> call parted and resize it.
- no partition, no need to call parted and continue next